app/views/layouts/application.html.erb

```diff
@@ -70,9 +70,7 @@
       <%= render 'ie_unsupported_warning' %>
     <% end %>
 
-    <% if current_user %>
-      <%= render 'front_end_admin_navbar' %>
-    <% end %>
+    <%= render 'front_end_admin_navbar' if can? :access_staff_functions %>
```

Comment:
honestly I think this would be more clear if you leave it three lines, instead of the trailing `if`, but no big deal.